rustls-libssl/src/x509.rs

```diff
@@ -43,6 +55,37 @@ impl OwnedX509Stack {
         self.raw
     }
 
+    /// Leak the first X509* to the caller.
+    ///
+    /// null is returned if the stack is empty, or itself null.
+    pub fn borrow_top_ref(&self) -> *mut X509 {
+        self.borrowed_item(0)
+    }
+
+    #[allow(dead_code)] // delete me later if unused
```

Comment:
It looks like this _is_ being used in `entry.rs` by `_SSL_CTX_ctrl` for the `cmd == SslCtrl::SetChain` case.

I think you can delete the `dead_code` allow here and on L81 for `item`